main/src/main/java/org/mobilitydata/gtfsvalidator/validator/StopTooFarFromTripShapeValidator.java

```diff
@@ -36,7 +47,7 @@
 /**
  * Validates: a {@link GtfsStop} is within a distance threshold for a trip shape.
  *
- * <p>Generated notice: {@link StartAndEndDateOutOfOrderNotice}.
+ * <p>Generated notice: {@link StopTooFarFromTripShapeNotice}.
```

Comment:
Could you also give time complexity of this algoritm?

Reply:
I've added notes on time complexity in https://github.com/MobilityData/gtfs-validator/pull/750/commits/e47ee19d3730fa1281782e89a75a156524f65641. I actually had to think about this a bit, and hopefully I've expressed it in a way that's understandable. Improvements welcome!